tests/phpunit/tests/test-rest-options.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+use WP_Syntex\Polylang\REST\API;
+
+class REST_Options_Test extends PLL_UnitTestCase {
+	/**
+	 * @var Spy_REST_Server
+	 */
+	private $server;
+
+	protected static $administrator;
+	protected static $author;
+
+	/**
+	 * @param PLL_UnitTest_Factory $factory
+	 */
+	public static function pllSetUpBeforeClass( PLL_UnitTest_Factory $factory ) {
+		parent::pllSetUpBeforeClass( $factory );
+
+		self::$administrator = self::factory()->user->create( array( 'role' => 'administrator' ) );
+		self::$author        = self::factory()->user->create( array( 'role' => 'author' ) );
+	}
+
+	public function set_up() {
+		parent::set_up();
+
+		add_action(
+			'pll_init',
+			function ( $polylang ) {
+				$polylang->rest = new API( $polylang->model );
+				add_action( 'rest_api_init', array( $polylang->rest, 'init' ) );
+			}
+		);
+
+		$this->pll_env = ( new PLL_Context_Rest() )->get();
+		$this->server  = $GLOBALS['wp_rest_server'];
+	}
+
+	public function tear_down() {
+		parent::tear_down();
```

Comment:
I think it is a good practice to call `parent::tear_down();` at the end.

Reply:
Indeed, copied from another class. Done in [e6cffa4](https://github.com/polylang/polylang/pull/1569/commits/e6cffa4c1150156ddb552a230dee799f10b1fc7f).